Sources/PackageModel/Package.swift

```diff
@@ -175,12 +180,13 @@ public struct PackageReference: JSONMappable, JSONSerializable, CustomStringConv
             "identity": identity,
             "path": path,
             "isLocal": isLocal,
-            ])
+            "isRoot": isRoot
```

Comment:
I think we store package references in `dependencies-state.json` so we should bump its schema version.

Reply:
I bumped the schema version to 3 and edited the `JSONMappable` conformance of `PackageReference` so we can set `supportedSchemaVersions` to `[2]`.